Providers/Modules/nxOMSAuditdPlugin/conf/auditd_plugin.conf

```diff
@@ -0,0 +1,29 @@
+<source>
+  @type auoms
+  tag oms.audit
+  path /var/opt/microsoft/omsagent/@@WORKSPACE_ID@@/run/auoms.socket
```

Comment:
I shouldn't have labelled this nit. It's only a string change, so I don't think you should re-test. You only have to change the instances of it in this file, line 8 in Providers/Modules/nxOMSAuditdPlugin/conf/auoms.conf, and the value of `WORKSPACE_ID_TMPL_STR` in your python files.

Since this is central to multi-homing compatibility, I want all the string replacement to be the same across our codebase.